member-service/src/main/java/com/hedvig/memberservice/query/MemberEventListener.java

```diff
@@ -271,4 +251,18 @@ void on(PickedLocaleUpdatedEvent e) {
     m.setPickedLocale(e.getPickedLocale());
     userRepo.save(m);
   }
+
+  @EventHandler
+  @Transactional
+  void on(MemberDeletedEvent e) {
+      if (userRepo.findById(e.getMemberId()).isPresent()) {
```

Comment:
Can we rename this repository, super confusing name now with the user concept 😄 
```suggestion
      if (memberRepository.findById(e.getMemberId()).isPresent()) {
```

Reply:
Sure, I can refactor it. This is very ancient legacy code and not something I've written. I just used the already existing variable.